test/unit/lib/service-test-2-window.html

```diff
@@ -0,0 +1,12 @@
+<script src="require.js"></script>
```

Comment:
This test fails on the current lib and passes with the change as it should. Please tell me if you have a better idea for the test that is less intrusive!